src/EnergyPlus/DataDaylighting.hh

```diff
@@ -95,88 +96,89 @@ namespace DataDaylighting {
         DElightDaylighting
     };
 
-    // Parameters for "Lighting Control Type"
-    enum class iLtgCtrlType
+    // Parameters for "Lighting Control Type" - these are the values expected by DElight
+    enum class LtgCtrlType
     {
+        Invalid = 0,
         Continuous = 1,
         Stepped = 2,
-        ContinuousOff = 3
+        ContinuousOff = 3,
```

Comment:
Why was a comma added here? Does unassigned = -1 apply here with num at the end? I see a lot of comments on that, and yeah it's a good practice, but is there any overhead for those enums where num is never used?

Reply:
Enumerations are compile-time constants, not runtime variables. There is no storage or other runtime overhead associated with them. [Here](https://github.com/NREL/EnergyPlus/wiki/enum-or-enum-class:-Either-Way,-Use-Them) is a more detailed explanation of the patterns we are trying to establish.